package.json

```diff
@@ -249,7 +249,8 @@
     "Jon Bretman <jon.bretman@gmail.com>",
     "Julian Connor <julian.connor@venmo.com>",
     "Jurko Gospodnetić <jurko.gospodnetic@pke.hr>",
-    "Karl Lindmark <karl.lindmark@ninetwozero.com>"
+    "Karl Lindmark <karl.lindmark@ninetwozero.com>",
+    "Matthew Amato <matt.amato@gmail.com>"
```

Comment:
Please revert this, we generate this list automatically on release via the git log